app/src/main/java/com/keystone/cold/ui/fragment/main/EthTxConfirmFragment.java

```diff
@@ -59,10 +62,12 @@
 import static com.keystone.cold.ui.fragment.setup.PreImportFragment.ACTION;
 
 public class EthTxConfirmFragment extends BaseFragment<EthTxConfirmBinding> {
-
+    private static final String VISITS_NAME = "visitsName";
+    private static final String SP_NAME = "EthTxConfirmFragment";
```

Comment:
Manage preferences in https://github.com/KeystoneHQ/Keystone-cold-app/blob/main/app/src/main/java/com/keystone/cold/Utilities.java

Reply:
Rename VISITS_NAME to a more business meaningful name.